analytic_tag_dimension/models/analytic.py

```diff
@@ -18,28 +19,51 @@ class AccountAnalyticDimension(models.Model):
         string='Analytic Tags')
 
     @api.model
-    def create(self, values):
-        if ' ' in values.get('code'):
-            raise ValidationError(_("Code can't contain spaces!"))
-        model_names = (
+    def get_model_names(self):
```

Comment:
Good enhancement. I have one question. As this module is used to extend dimension in other module, i..e, sale.order.line, purchase.order.line
https://github.com/ecosoft-odoo/account-analytic/blob/12.0-mig-analytic_tag_dimension_sale_warning/analytic_tag_dimension_sale_warning

Will this PR has affect on those?

Reply:
I think that without this PR, the linked module don't have a inheritable way to create dimension columns in mentioned models, if you need.
So this PR beneficially affects if you extend `get_model_names` and does not affect at all if you don't.